diff --git a/usr/share/jolla-settings/pages/mobilenetworking/SimMobileNetworkSettings.qml b/usr/share/jolla-settings/pages/mobilenetworking/SimMobileNetworkSettings.qml
index a6a6fa8..302eed0 100644
--- a/usr/share/jolla-settings/pages/mobilenetworking/SimMobileNetworkSettings.qml
+++ b/usr/share/jolla-settings/pages/mobilenetworking/SimMobileNetworkSettings.qml
@@ -226,24 +226,33 @@ Column {
         menu: ContextMenu {
             id: networkModeMenu
             MenuItem {
+                readonly property string tech: "any"
+                text: "Any"
+                visible: radioSettings.availableTechnologies.indexOf(tech) >= 0
+            }
+            MenuItem {
                 readonly property string tech: "lte"
-                //: Network mode settings ComboBox item for preferring 4G/LTE
-                //% "Prefer 4G"
-                text: qsTrId("settings_network-me-network_mode_prefer_4G")
+                text: "4G"
+                visible: radioSettings.availableTechnologies.indexOf(tech) >= 0
+            }
+            MenuItem {
+                readonly property string tech: "umtslte"
+                text: "4G+3G"
                 visible: radioSettings.availableTechnologies.indexOf(tech) >= 0
             }
             MenuItem {
                 readonly property string tech: "umts"
-                //: Network mode settings ComboBox item for prefering 3G/umts
-                //% "Prefer 3G"
-                text: qsTrId("settings_network-me-network_mode_prefer_3G")
+                text: "3G"
+                visible: radioSettings.availableTechnologies.indexOf(tech) >= 0
+            }
+            MenuItem {
+                readonly property string tech: "gsmumts"
+                text: "3G+2G"
                 visible: radioSettings.availableTechnologies.indexOf(tech) >= 0
             }
             MenuItem {
                 readonly property string tech: "gsm"
-                //: Network mode settings ComboBox item for 2G/gsm only
-                //% "2G only"
-                text: qsTrId("settings_network-me-network_mode_2G_only")
+                text: "2G"
                 visible: radioSettings.availableTechnologies.indexOf(tech) >= 0
             }
         }
